src/form-builder/components/FormBuilder.jsx

```diff
@@ -183,20 +183,21 @@ export default class FormBuilder extends Component {
     Promise.all(fileParsePromises).then(() => self.processForms(formJsons));
   }
 
-  processForms(formJsons) {
+  async processForms(formJsons) {
     const formsValidationPromises = [];
     const self = this;
-    formJsons.forEach(form => {
+    for (const form of formJsons) {
```

Comment:
@supriyatw @venkybavisetti Why is the forEach changed to for loop any reason?

Reply:
Apparently, we cannot use async with foreach, hence added for loop.